.circleci/config.yml

```diff
@@ -1098,20 +1251,7 @@ jobs:
           event: pass
           template: success_tagged_deploy_1
 
-  wait-for-binaries:
```

Comment:
Is this being removed fully, or is there another step that will take care of this?

Reply:
It was a duplicated unused job